.vscode/settings.json

```diff
@@ -8,5 +8,5 @@
     {"directory": "packages/jest-gas-reporter", "changeProcessCWD": true},
     {"directory": "packages/nitro-protocol", "changeProcessCWD": true}
   ],
-  "editor.formatOnSave": true
+  "editor.formatOnSave": false
```

Comment:
Could you just remove this? That way any settings we have set locally still apply.